### ISSUE TITLE
Issue with EasyEDA Gerbers

### ISSUE THREAD

**Issue by kasbah** [mcous/gerber-parser#21]
This [Gerber from EasyEDA is causing an issue](https://cloud.monostable.co.uk/Gerber_drill.GBL) (from @squaresolid [tiger_biosynth](https://github.com/squaresolid/tiger_biosynth)). I guess the title block is not being parsed? Interestingly, it works fine without the zero-supression detection :/

```
/tmp/gerber-to-svg/node_modules/gerber-parser/lib/parse-coord.js:25
    throw new Error('cannot parse coordinate with format undefined')
    ^

Error: cannot parse coordinate with format undefined
```

**Comment by mcous** [comment]
This looks like the same issue as #19 (presumed fixed with #20, v1.0.7). Is this error occurring with the latest gerber-parser?

**Comment by kasbah** [comment]
Oops, yes it's the same issue. I was using the zero-suppression branch.
